core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1224,6 +1224,9 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     case "BITNOT":
       unparseBitNotFunction(writer, call);
       break;
+    case "SHIFTRIGHT":
+      unparseShiftRight(writer, call);
```

Comment:
can we remove these two functions unparseShiftRight(writer, call) & unparseShiftLeft(writer, call) and can directly use
unparseShiftLeftAndShiftRight(writer, call, true/false) calls respectively.

Reply:
done

Reply:
looks good.